README.md

```diff
@@ -3,7 +3,7 @@ Caffeine
 
 **CoArray Fortran Framework of Efficient Interfaces to Network Environments**
 
-Caffeine is a parallel runtime library that aims to support Fortran compilers with a programming-model-agnostic application binary interface (ABI) to various communication libraries.  Current work is on supporting the ABI with the [GASNet-EX] exascale-ready networking middleware.  Future plans include support for an alternative Message Passing Interface ([MPI]) back end.
+Caffeine is a parallel runtime library that aims to support Fortran compilers with a programming-model-agnostic application programming interface (API) to various communication libraries.  Current work is on supporting the API with the [GASNet-EX] exascale-ready networking middleware.  Future plans include support for an alternative Message Passing Interface ([MPI]) back end.
```

Comment:
@ktras I think we've now moved on from "API" to "PRIF" or just "interface".   Please adjust accordingly.

Reply:
Done